x-pack/test/functional/apps/maps/import_geojson/file_indexing_panel.js

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import path from 'path';
+import uuid from 'uuid/v4';
+
+export default function ({ getService, getPageObjects }) {
+  const PageObjects = getPageObjects(['maps', 'common']);
+  const testSubjects = getService('testSubjects');
+  const log = getService('log');
+
+  const IMPORT_FILE_PREVIEW_NAME = 'Import File';
+  const FILE_LOAD_DIR = 'test_upload_files';
+  const DEFAULT_LOAD_POINT_FILE_NAME = 'point.json';
+  let indexName;
+
+  const indexPoint = async () => await loadFileAndIndex(DEFAULT_LOAD_POINT_FILE_NAME);
+  const indexFileByName = async fileName => await loadFileAndIndex(fileName);
+
+  async function loadFileAndIndex(loadFileName) {
+    await PageObjects.maps.clickAddLayer();
+    // Verify panel page element is open
+    const panelOpen = await PageObjects.maps.isLayerAddPanelOpen();
```

Comment:
This check is not needed. The function will fail in `selectGeoJsonUploadSource` if the panel is not opened.

expects should be used to show what the test is verifying. The test is not verifying the panel is open. This is just part of the set-up